modules/client/src/main/java/org/apache/ignite/internal/client/table/AbstractClientView.java

```diff
@@ -50,6 +56,40 @@ abstract class AbstractClientView<T> implements CriteriaQuerySource<T> {
         this.tbl = tbl;
     }
 
+    /**
+     * Get index mapping from result set to schema.
+     *
+     * @param columns Columns to map.
+     * @param metadata Metadata for query results.
+     * @param startInclusive The first index to cover.
+     * @param endExclusive Index immediately past the last index to cover.
+     * @return Index mapping.
+     */
+    static List<Integer> indexMapping(
+            ClientColumn[] columns,
+            int startInclusive,
+            int endExclusive,
+            @Nullable ResultSetMetadata metadata
```

Comment:
Why it marked as Nullable if as the first action here is checking on null and throws an exception?